contracts/SpokePool.sol

```diff
@@ -447,7 +454,7 @@ abstract contract SpokePool is
         int64 relayerFeePct,
         uint32 quoteTimestamp,
         bytes memory message,
-        uint256 maxCount
+        uint256 // maxCount. Deprecated.
```

Comment:
Should we still keep this as a named parameter - even if we're ignoring it?

Reply:
that would trigger the linter since its unused, why keep it named?